packages/schemas/src/Components/Utilities/Get.php

```
@@ -31,4 +35,57 @@ public function __invoke(string | Component $path = '', bool $isAbsolute = false
 
         return $component->getState();
     }
+
+    public function string(string $key, bool $isAbsolute = false): string
+    {
+        try {
+            return (string) ($this($key, $isAbsolute) ?? '');
```

I think this should throw an error if its not a stringable value, so remove the `try`